Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -14,29 +14,16 @@ extension Method {
     /// Concatenates all function parameters in a string
     var methodParameters: String {
         parameters.map { parameter in
-            guard parameter.typeName.isOpaqueType && parameter.typeName.isOptional else {
-                return parameter.asSource
-            }
-
-            /// This will dissect the parameter containing an optional opaque type and put brackets around it
-            ///
-            /// Example of parameter.asSource result:
-            /// `parameter: any OpaqueType?`
-            ///
-            /// Result of function will be:
-            /// `parameter: (any OpaqueType)?`
+            let parameterName = parameter.name
```

Comment:
We might want to upgrade this to make use of the external name. (paramter.argumentLabel)
Something like 
```swift
[paramater.argumentLabel, parameter.name].compactMap { $0 }.joined(separator: " ")
```
But the .argumentLabel and .name can be the same, so we should add filter to only have distinct strings